From be6fb589406d3b1777d9a07c8d2ee8e9a59e0a82 Mon Sep 17 00:00:00 2001
From: pei-jiankang <peijiankang@kylinos.cn>
Date: Wed, 11 May 2022 13:54:44 +0800
Subject: [PATCH] modify compile error

---
 plugins/xrandr/xrandr-output.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/plugins/xrandr/xrandr-output.cpp b/plugins/xrandr/xrandr-output.cpp
index 4162547..624db24 100644
--- a/plugins/xrandr/xrandr-output.cpp
+++ b/plugins/xrandr/xrandr-output.cpp
@@ -363,9 +363,9 @@ void xrandrOutput::readInOutputs(KScreen::ConfigPtr config, const QVariantList &
         auto replicationSource = nullptr;//control.getReplicationSource(output);
         if (replicationSource) {
             //output->setPos(replicationSource->pos());
-            //output->setLogicalSize(replicationSource->logicalSize());
+            //output->setExplicitLogicalSize(replicationSource->logicalSize());
         } else {
-            output->setLogicalSize(QSizeF());
+            output->setExplicitLogicalSize(QSizeF());
         }
     }
 
-- 
2.33.0

