From 1820a7ccc55b03c3629b446eb19f55e75962982c Mon Sep 17 00:00:00 2001
From: tanyulong2021 <tanyulong@kylinos.cn>
Date: Thu, 4 Aug 2022 11:37:51 +0800
Subject: [PATCH] don't depend xserver-xorg-input-synaptics on s390x

---
 debian/control | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/control b/debian/control
index c98dea1..85ea207 100755
--- a/debian/control
+++ b/debian/control
@@ -45,7 +45,7 @@ Depends: mate-desktop-common,
          imwheel,
          ukui-settings-daemon-common (= ${source:Version}),
          x11-xserver-utils,
-         xserver-xorg-input-synaptics,
+         xserver-xorg-input-synaptics [!s390x],
          ${misc:Depends},
          ${shlibs:Depends},
 Description: daemon handling the UKUI session settings
-- 
2.33.0

